media-sound/teamspeak-server-bin/teamspeak-server-bin-3.0.13.8.ebuild

```diff
@@ -12,7 +12,7 @@ SRC_URI="
 	x86? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/teamspeak3-server_linux_x86-${PV}.tar.bz2 )"
 
 SLOT="0"
-LICENSE="teamspeak3 GPL-2"
+LICENSE="teamspeak3"
```

Comment:
I don't think this is relevant in this commit.

Reply:
Yes, fixed.